src/index.html

```diff
@@ -16,17 +16,29 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+  <body>
+    <form
+      class="searchbar"
+      data-qa="search-form"
+    >
+      <input
+        class="searchbar__input searchbar__input-big"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
+    <form
+      data-qa="search-input"
```

Comment:
The `data-qa` attribute for the second form should be reviewed. Ensure it matches the task requirements for identifying the search input.